tests/tasks/test_binary.py

```diff
@@ -0,0 +1,50 @@
+"""
+Copyright (c) 2021 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+import pytest
+
+from atomic_reactor import source
+from atomic_reactor.tasks import common
+
+GIT_URI = "git://example.org/namespace/repo"
+
+
+class TestBinaryBuildTaskParams:
```

Comment:
We probably don't need any additional unit tests for the `binary` module, it doesn't implement anything, just declares plugin lists

Reply:
removed it